File: cross_platform/OHaraEdward_CrossPlatform/app/src/main/java/com/ohara/oharaedward_crossplatform/dataModel/UserData.java
package com.ohara.oharaedward_crossplatform.dataModel;

import java.io.Serializable;


//Edward O'Hara
//UserData
public class UserData implements Serializable{

    //stored properties
    public String firstName;
    public String lastName;
    public String email;
    public String password;//never store password in firebase, used for login w/ firebase
    public String uid;

    //firebase default constructor
    public UserData(){
    }

    //constructor
    public UserData(String firstName, String lastName, String email, String password) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.password = password;
    }
}

File: cross_platform/OHaraEdward_iOS_CrossPlatform/OHaraEdward_iOS_CrossPlatform/UserData.swift
//
//  UserData.swift
//  OHaraEdward_iOS_CrossPlatform
//
//  Created by Scott O'Hara on 4/4/17.
//  Copyright © 2017 Scott O'Hara. All rights reserved.
//

import Foundation

class UserData{
    
    //stored properties
    var firstName: String
    var lastName: String
    var email: String
    var password: String?//never store password in firebase
    var uid: String
    
    //init user properties
    init(firstName: String, lastName: String, email: String, uid: String) {
        self.firstName = firstName
        self.lastName = lastName
        self.email = email
        self.uid = uid
    }
}
